tests/code-splitting/subtests/multi-entry/webpack/index.md

```diff
@@ -1,12 +1,12 @@
 ---
-result: partial
+result: pass
```

Comment:
> I wonder - if webpack were to _warn_ in cases where modules are being duplicated as a result of `runtimeChunk` not being set to `"single"`, that would feel like a very solid `pass` upgrade here. Do you think that would be feasible?

I can look into that. It's a bit tricky as Multiple Entries and `runtimeChunk: false` is not incorrect. The duplication is only a problem in the edge case when using Multiple Entries on a single page. This is actually not the normal way. Usually you should have a 1 to 1 mapping from HTML to entrypoint. Multiple Entrypoints per Page is only recommended when you need to decide at (server) runtime which entrypoints should be loaded in a page.
And there is also `dependOn` for most cases where you want to put Multiple Entries on a single page. This handles the runtime correctly.

Reply:
I bet we could add a note here that correct use of `dependsOn` is required for multiple entries on a single page to share chunks. That feels more like a pass to me than pointing folks at `runtimeChunk:single`.

Reply:
I bet we could add a note here that correct use of `dependsOn` is required for multiple entries on a single page to share chunks. That feels more like a pass to me than pointing folks at `runtimeChunk:single`.

Reply:
@sokra 

> The duplication is only a problem in the edge case when using Multiple Entries on a single page. This is actually not the normal way. Usually you should have a 1 to 1 mapping from HTML to entrypoint.

That hasn't been my experience in larger companies, where multiple teams are responsible for different parts of a page. It's often true for smaller one-team projects though, but not always. Even on https://bundlers.tooling.report we have two entry points, one 'async' and one 'defer'.

@developit I'm still tempted to go 'partial' here because you have to manually opt out of a gotcha.

Reply:
@jakearchibald Yes, that is my experience as well - using webpack to bundle a bunch of widgets (with shared deps) that are then embedded into pages generated by PHP.